src/visualization/utilities.jl

```diff
@@ -481,6 +481,27 @@ function get_unstructured_data(u, solution_variables, mesh, equations, solver, c
     return unstructured_data
 end
 
+function get_unstructured_data(func::Function, solution_variables, mesh, equations,
```

Comment:
Don't you need the `where {Function}` here such that this makes sense?

Reply:
`Function` is a data type. So this can be used for dispatching. As an alternative, one could write `func::F` and then `where {F <: Function}`, but as far as I know this does not have a benefit compared to the `func::Function` version.
I think, generally it is discouraged to dispatch on `Function` as, e.g., callable structs do not fall under this type. However, I didn't see an easy alternative. I also tried to do the dispatch on the other function, which takes an array, but that lead to some method ambiguities. But I'm open for improvements.